packages/angular-table/src/public-api.ts

```diff
@@ -0,0 +1,7 @@
+/*
+ * Public API Surface of angular-table
+ */
+
+export * from '@tanstack/table-core'
+export * from './lib/flexrender.directive'
+export * from './lib/index'
```

Comment:
Where are those files ?

Reply:
Actually got messed up by this commit : https://github.com/TanStack/table/pull/5326/commits/91117bcaa524f09076a8dfbfd8f6f2340c29557f

Reply:
@KevinVandy @JeanMeche @Lord-AY 
export * from './lib/flexrender.directive'
export * from './lib/index'
moved  above files from lib folder to src folder, updated examples